pulsar_spectra/catalogue.py

```diff
@@ -273,7 +273,7 @@ def collect_catalogue_fluxes(only_use=None, exclude=None, query=None):
 
     # Add the antf to the cataogues
     antf_dict = all_flux_from_atnf(query=query)
-    antf_incorrect_refs = ["Zhao_2019", "Mignani_2017", "Stovall_2015"]
+    antf_incorrect_refs = ["Zhao_2019", "Mignani_2017", "Stovall_2015", "Bell_2016", "Lorimer_1995"]
```

Comment:
Can you please send me a pulsar or two that you noticed these errors. I'll have a look into them and decide if we have to notify ATNF or edit our code.

Reply:
<img width="1019" alt="dups_1" src="https://user-images.githubusercontent.com/96163140/168100066-ad3ec437-ff5b-492c-a803-68f55ee08a5a.png">
<img width="1161" alt="dups_5" src="https://user-images.githubusercontent.com/96163140/168100110-f82e7f90-cf58-4fa9-8f4e-833618ebbe14.png">
<img width="1019" alt="dups_4" src="https://user-images.githubusercontent.com/96163140/168100123-1fc666ff-403a-4fbb-ac65-24204627bfa6.png">
<img width="1019" alt="dups_3" src="https://user-images.githubusercontent.com/96163140/168100128-627eb987-4a3e-4b69-b63e-66311b3c6bb0.png">
<img width="1019" alt="dups_2" src="https://user-images.githubusercontent.com/96163140/168100135-d697a2c7-f012-4a96-9f8a-0d82c1ce11bf.png">

Reply:
I've checked and 

Lorimer_1995 ATNF data is correct but were scaled to by their spectral index for the ATNF frequencies

Bell_2016 ANTF data flux uncertainties are off by an order of magnitude